addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_materials.py

```diff
@@ -55,6 +56,8 @@ def gather_material(blender_material, mesh_double_sided, export_settings):
         pbr_metallic_roughness=__gather_pbr_metallic_roughness(blender_material, orm_texture, export_settings)
     )
 
+    export_user_extensions('gather_materials_hook', export_settings, material, blender_material)
```

Comment:
`gather_material_hook`? (With no `s`.) Since the function is called `gather_material`?

Reply:
Done